tests/size.rs

```diff
@@ -0,0 +1,13 @@
+#[test]
+#[cfg(not(debug_assertions))]
+fn assert_size() {
+    use std::mem::size_of;
+
+    use crate::sys;
```

Comment:
This shouldn't be possible as `sys` is not public.

Reply:
This doesn't get tested on the CI since it always has debug assertions, so can you can check with `cargo test --release`.

Related #1035.